go.mod

```
@@ -1,6 +1,6 @@
 module github.com/strangelove-ventures/cosmos-operator
 
-go 1.21
+go 1.23.0
```

Let's do either

```suggestion
go 1.23
```

Or latest
```suggestion
go 1.23.2
```